cli/src/commands/extensions/mod.rs

```diff
@@ -73,8 +73,17 @@ pub async fn handle_extensions(matches: &ArgMatches) -> CommandResult {
 
 /// Handle the `extension add` subcommand path.
 /// Add the extension from the specified path.
-async fn handle_add_extension(path: PathBuf) -> CommandResult {
-    let extension = Extension::try_from(path)?;
+async fn handle_add_extension(path: &str) -> CommandResult {
+    // NOTE: Extension installation without slashes is reserved for the marketplace.
+    if !path.contains('/') {
```

Comment:
With the force pushes to this PR, it is not possible to see the history...but didn't this line get changed to:
`if !path.contains('/') && !path.contains('\\') {`
at one point? Did it go back?

Granted, the error message will tell the user the expected format to use, but this error message is _much_ more likely to present itself to Windows users...which may be considered unwelcoming. Perhaps a more nuanced discussion has already taken place with @kylewillmon, but he is out this week and any decisions regarding path separator support should be documented here and on the issue (#356)...which currently only says:

> After discussion, we have decided to require slashes in the argument for local installs. I have updated the title and description accordingly.

...but not which slashes or if only one type of slash will be accepted.

Reply:
Yeah, good catch. This is what happens when you use the GitHub "apply suggestion" feature once every other year and then completely forget about it when you rebase the commit.